e2e/test-specs.yml

```diff
@@ -0,0 +1,94 @@
+description: |
+  End-to-end tests for k8s integration
+
+custom_test_key: k8s.clusterName
+
+scenarios:
+  - description: |
+      This scenario will verify that metrics from a k8s Cluster are correctly collected.
+    before:
+      - helm upgrade --install ${SCENARIO_TAG}-resources ./charts/e2e-resources
+      - helm upgrade --install ${SCENARIO_TAG} newrelic/nri-bundle --values e2e-values.yml --set global.licenseKey=${LICENSE_KEY} --set global.cluster=${SCENARIO_TAG}
+    after:
+      - kubectl describe pods -l app=newrelic-infrastructure
+      - kubectl logs -l app=newrelic-infrastructure
+      - helm delete ${SCENARIO_TAG}-resources
+      - helm delete ${SCENARIO_TAG}
+    tests:
+      nrqls: []
```

Comment:
We would need something like:
```
 select *  from (SELECT uniqueCount(k8s.podName) as a  FROM Metric  where k8s.clusterName = 'e2e-2bbbf04-glroj' and k8s.namespaceName = 'nr-e2e-2bbbf04-glroj') where a = 8
```
that would not work because k8s.namespaceName = 'nr-e2e-2bbbf04-glroj' is not added in the inner query

We could add something checking each pod/deployment/sample one by one, but it would be really verbose and I think that it would not worth it.

I would add this in the next iteration of the action, or when querying  with DM cover all use-cases